packages/integration-karma/test/light-dom/synthetic-shadow/index.spec.js

```diff
@@ -25,7 +25,27 @@ describe('Light DOM + Synthetic Shadow DOM', () => {
             expect(nodes.p.assignedSlot).toEqual(nodes.slot);
         });
         it('childNodes', () => {
+            // TreeWalker is just a convenient way of getting text nodes without using childNodes
+            const textNodes = {};
+            const walker = document.createTreeWalker(elm, NodeFilter.SHOW_TEXT);
+            let node;
+            while ((node = walker.nextNode())) {
+                textNodes[node.wholeText] = node;
+            }
             expect(Array.from(nodes.slot.childNodes)).toEqual([]);
+            expect(Array.from(nodes.p.childNodes)).toEqual([
+                textNodes['I am an assigned element.'],
+            ]);
+            expect(Array.from(nodes.consumer.childNodes)).toEqual([
+                nodes.p,
+                textNodes['I am an assigned text.'],
+            ]);
+            expect(Array.from(elm.childNodes)).toEqual([nodes.consumer]);
+            expect(Array.from(nodes['consumer.shadowRoot'].childNodes)).toEqual([
+                nodes.pInShadow,
+                nodes.slot,
+            ]);
+            expect(Array.from(nodes.slot)).toEqual([]);
```

Comment:
These tests are maybe a bit excessive, but I just wanted to make sure we were covering all the bases.